website/docs/docs/build/sl-getting-started.md

```diff
@@ -91,46 +154,85 @@ group by metric_time_day, country  -- dimensions
   ```
 :::
 
-## Create your metrics
+## Define metrics
+
+Now that you've created your first semantic model, it's time to define your first metric! MetricFlow supports different metric types like [simple](/docs/build/simple), [ratio](/docs/build/ratio), [cumulative](/docs/build/cumulative), and [derived](/docs/build/derived). It's recommended that you read the [metrics overview docs](https://docs.getdbt.com/docs/build/metrics-overview) before getting started. 
 
-Now that you've created your first semantic model, it's time to define your first metric. MetricFlow supports different metric types like [simple](/docs/build/simple), [ratio](/docs/build/ratio), [cumulative](/docs/build/cumulative), and [derived](/docs/build/derived). You can define metrics in the same YAML files as your semantic models, or create a new file.
+1. You can define metrics in the same YAML files as your semantic models, or create a new file. If you want to create your metrics in a new file, we recommend creating another directory called `/models/metrics`. The file structure for metrics can become more complex from here if you need to further organize your metrics, for example by data source or business line. 
```

Comment:
With instructional material, especially for a new feature, it can instill more confidence in the steps and better establish best practices to avoid "we recommend" and tell them firmly how to do it. It also avoids the potential mess of having to answer for how to do it alternatively. 
```suggestion
1. You can define metrics in the same YAML files as your semantic models, or create a new file. If you want to create your metrics in a new file, create another directory called `/models/metrics`. The file structure for metrics can become more complex from here if you need to further organize your metrics, for example, by data source or business line. 
```